### DISCUSSION TITLE
Welcome to deck-of-cards Discussions!

### DISCUSSION THREAD

**Discussion by pakastin** [pakastin/deck-of-cards#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JinJinJinT** [comment]
Hello! My name is Jin and I'm an Electrical Engineering major at the University of Washington. I'm trying to make a version of the game BigTwos for my friends and me to play online. Does anyone know where I could gain access to the newer repository of the card deck? Thanks!

**Reply by ccj242** [reply]
Very interested in the new repo, any updates?

**Comment by iamn1ck** [comment]
I've been following this project for a while, but still waiting for the multiplayer source code. There is also a project I'd like to build.
